app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html

```diff
@@ -22,7 +22,7 @@
 @import uk.gov.hmrc.nisp.controllers.auth.NispUser
 @import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
 
-@(nino: String, spSummary: SPSummaryModel, authenticationProvider: String, isPertaxUrl:Boolean
+@(nino: String, spSummary: SPSummaryModel, authenticationProvider: String, confidenceLevel: String, authProviderGA: Option[String], isPertaxUrl:Boolean
```

Comment:
Somehow the value passed is the correct one i.e IDA or GGW. I am also thinking if you are in the account page, there would always be an auth provider, don't you think?